### PR TITLE
Add formula for installing iBazel

### PR DESCRIPTION
**Pull request by achew22**

### CODE REVIEW

**Comment by Globegitter** [comment]
@laurentlb @dslomov Anything blocking this from getting merged, would make it considerably easier for our devs on mac to install ibazel.

**Comment by laurentlb** [comment]
cc @buchgr @philwo

**Comment by dslomov** [comment]
I want @philwo or @buchgr to take a look as well

File: ibazel.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class Ibazel < Formula
+  desc 'IBazel is a tool for building Bazel targets when source files change.'
+  homepage 'https://github.com/bazelbuild/bazel-watcher'
+  version '0.5.0'
+  url "https://codeload.github.com/bazelbuild/bazel-watcher/tar.gz/v0.5.0"
+  sha256 '6b727c60a807b56ebcf7b76c1002657bd93ad9199a2caa1c7ccd1a7eff4a4659'
+
+  bottle :unneeded
+
+  depends_on "bazel" => :install
```
Comment (buchgr) [inline ibazel.rb]: I think this depends on the bazel in homebrew core no? If so that's not what we would want.

File: ibazel.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class Ibazel < Formula
+  desc 'IBazel is a tool for building Bazel targets when source files change.'
+  homepage 'https://github.com/bazelbuild/bazel-watcher'
+  version '0.5.0'
+  url "https://codeload.github.com/bazelbuild/bazel-watcher/tar.gz/v0.5.0"
+  sha256 '6b727c60a807b56ebcf7b76c1002657bd93ad9199a2caa1c7ccd1a7eff4a4659'
```
Comment (philwo) [inline ibazel.rb]: AFAIK the hash might occasionally change when GitHub updates their backend that generates these files. We had this happen once in Bazel: https://github.com/bazelbuild/bazel/issues/3722

File: ibazel.rb (hunk 1 above)
Comment (achew22) [reply ibazel.rb]: How would I specify the tap based version?

File: ibazel.rb (hunk 2 above)
Comment (achew22) [reply ibazel.rb]: Is there any kind of CI running on this? Could we get one set up to attempt to `brew install`/`brew test` everything specified in this repo in a docker container or something to make sure we don't have silent breakages?

File: ibazel.rb (hunk 1 above)
Comment (Globegitter) [reply ibazel.rb]: I have no idea about this, but maybe it is `bazelbuild/tap/bazel` as this is also the `brew install` command that has be used to install bazel from this tap.

**Pull request merged**